charts/karmada/templates/post-install-job.yaml

```diff
@@ -38,6 +38,7 @@ spec:
       nodeSelector:
       {{- toYaml . | nindent 8 }}
       {{- end }}
+      serviceAccountName: {{ $name }}-pre-job
```

Comment:
Using the serviceAccount  `{{ $name }}-pre-job` in the post hook job doesn't seem very reasonable. can we modify `{{ $name }}-pre-job` to `{{ $name }}-hook-job`?

Reply:
I also want to change a name